finetune.py

```diff
@@ -65,21 +108,63 @@ def get_batch(data_iterator):
     data_b = tensor_parallel.broadcast_data(keys, data, datatype)
 
     # Unpack.
-    tokens_ = data_b['text'].long()
-    labels = tokens_[:, 1:].contiguous()
-    tokens = tokens_[:, :-1].contiguous()
-
-    # Get the masks and postition ids.
-    attention_mask, loss_mask, position_ids = get_ltor_masks_and_position_ids(
-        tokens,
-        tokenizer.eod,
-        args.reset_position_ids,
-        args.reset_attention_mask,
-        args.eod_mask_loss)
+    tokens = data_b["text"]
+    labels = tokens[:, 1:].contiguous()
+    tokens = tokens[:, :-1].contiguous()
+    if args.data_type == "gpt":
+
+        # Get the masks and postition ids.
```

Comment:
typo

Reply:
Mmm I can't see any typo. Maybe it was in another line?